### DISCUSSION TITLE
Jetbrains Chat Support

### DISCUSSION THREAD

**Discussion by imaskillissue** [community/community#55520] [Copilot Conversations]
I got in to copilot chat unfortunately it's doesn't have Jetbrains support. I won't switch from my Jetbrains IDEs over to VSC just to use the chat feature frequently especially for Java Kotlin projects.

**Comment by 1366613** [comment]
Nothing, not even Copilot Chat can convince anybody to switch to VS Code.
Can't wait to see when this feature is available as a JetBrains extension.

**Reply by sakuthush** [reply]
Jetbrains extension integration was hectic.. Very dual authentication type and it asks for subscription money over again.

**Reply by Faq** [reply]
@sakuthush copilot is not for free, so it will ask to pay. Only previews and test period is for free.

**Reply by sakuthush** [reply]
I had a monthly paid sub...

**Reply by Faq** [reply]
Then this is not the place to report bug ;)
